components/thumbnail/index.css

```diff
@@ -127,229 +112,108 @@ governing permissions and limitations under the License.
+	block-size: calc(var(--spectrum-thumbnail-size) - ((var(--mod-thumbnail-layer-border-width-inner, var(--spectrum-thumbnail-layer-border-width-inner)))) * 2);
 	outline-style: solid;
-	outline-color: var(
-		--highcontrast-thumbnail-layer-border-color-inner,
-		var(
-			--mod-thumbnail-layer-border-color-inner,
-			var(--spectrum-thumbnail-layer-border-color-inner)
-		)
-	);
-	outline-width: calc(
-		var(
-				--mod-thumbnail-layer-border-width-inner,
-				var(--spectrum-thumbnail-layer-border-width-inner)
-			) -
-			var(
-				--mod-thumbnail-layer-border-width-outer,
-				var(--spectrum-thumbnail-layer-border-width-outer)
-			)
-	);
+	outline-color: var(--highcontrast-thumbnail-layer-border-color-inner, var(--mod-thumbnail-layer-border-color-inner, var(--spectrum-thumbnail-layer-border-color-inner)));
+	outline-width: calc(var(--mod-thumbnail-layer-border-width-inner, var(--spectrum-thumbnail-layer-border-width-inner)) - var(--mod-thumbnail-layer-border-width-outer, var(--spectrum-thumbnail-layer-border-width-outer)));
 }
 /* stylelint-enable declaration-block-no-redundant-longhand-properties */
 
 .spectrum-Thumbnail-image-wrapper {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  inline-size: 100%;
-  block-size: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	inline-size: 100%;
+	block-size: 100%;
 }
 
 .spectrum-Thumbnail-image {
-  position: relative;
-  max-block-size: 100%;
-  max-inline-size: 100%;
-  z-index: 1;
+	position: relative;
+	max-block-size: 100%;
+	max-inline-size: 100%;
+	z-index: 1;
 }
 
 .spectrum-Thumbnail--cover {
-  .spectrum-Thumbnail-image {
-    block-size: 100%;
-    inline-size: 100%;
-    object-fit: cover;
-    object-position: center;
-  }
+	.spectrum-Thumbnail-image {
+		block-size: 100%;
+		inline-size: 100%;
+		object-fit: cover;
+		object-position: center;
+	}
 }
 
 .spectrum-Thumbnail-background {
-  position: absolute;
-  z-index: 0;
-  inset-inline-start: 0;
-  inset-block-start: 0;
-  inset-inline-end: 0;
-  inset-block-end: 0;
-  block-size: 100%;
-  inline-size: 100%;
-  background-size: cover;
-  background-position: center center;
-  border-radius: var(--mod-thumbnail-border-radius, var(--spectrum-thumbnail-border-radius));
+	position: absolute;
+	z-index: 0;
+	inset-inline-start: 0;
+	inset-block-start: 0;
+	inset-inline-end: 0;
+	inset-block-end: 0;
+	block-size: 100%;
+	inline-size: 100%;
+	background-size: cover;
+	background-position: center center;
+	border-radius: var(--mod-thumbnail-border-radius, var(--spectrum-thumbnail-border-radius));
 }
 
 /* Windows High Contrast Mode */
 @media (forced-colors: active) {
-  .spectrum-Thumbnail {
-    /* Allow checkerboard pattern to be visible. */
-    forced-color-adjust: none;
+	.spectrum-Thumbnail {
+		/* Allow checkerboard pattern to be visible. */
```

Comment:
I found that confusing — I'm inclined to make the checkerboard changes as a separate PR since they'd be in that component even if they impact the thumbnail. I'm open to combining them or proceeding differently though. ✨

Reply:
I had an idea for this- what we if added `customStyles` to the [opacity checkerboard in the thumbnail template](https://github.com/adobe/spectrum-css/blob/273bc2ea797fff70f7c7cf36fa19a00f5eef447d/components/thumbnail/stories/template.js#L113)? 

```
customStyles: {
	"--spectrum-opacity-checkerboard-size": "var(--spectrum-thumbnail-opacity-checkerboard-square-size)"
}
```

I tried to avoid the other layer thumbnail's checkerboard (since I agree- let's do those token changes with treeview), and I think this is just changing the opacity checkerboard size for the default thumbnail only.

The opacity checkerboard CSS doesn't have anything in it currently that scopes to the thumbnail, which was why I thought maybe the intention of that token is to override whatever is going on in the default opacity checkerboard component. Passing `size: "s"` to the opacity checkerboard works too, but it's not technically using the `thumbnail-opacity-checkerboard-square-size` token. 

What do you think?

Reply:
Great idea! Just pushed up an update. 🚀